maven-model/src/main/mdo/maven.mdo

```diff
@@ -1087,10 +1087,8 @@
           <version>3.0.0+</version>
           <description>
             <![CDATA[
-            The version requirement of the dependency, e.g. <code>3.2.1</code>. The actual version will be resolved based on the usage context.
-            Version requirement can also be specified as a range of versions, e.g. <code>[3.2.0,)</code>. This is discouraged as it may break <i>predictability</i> of resolved version.
-            See <a href="https://s.apache.org/dependency-version">dependency version requirement documentation</a>
-            and <a href="https://s.apache.org/transitive-dependencies-resolution">transitive dependencies resolution</a> for more details.
+            The version of the dependency such as <code>3.2.1</code>. This can also be
```

Comment:
This goes beyond removing references to Maven 2 and changes the wording and the semantics here.  I'm really not sure that blurring the differences between a `dependency constraint (or coordinates)` and the `resolved dependency' is a good idea.  We did not finish the discussion, but this goes opposite to the existing naming:
- https://maven.apache.org/ref/4-LATEST/api/maven-api-core/dependency-info.html
- https://maven.apache.org/ref/4-LATEST/api/maven-api-core/apidocs/org/apache/maven/api/package-summary.html

Reply:
Reverted most of this, with edits for corrections. I'm not sure it says as much as you're reading into it though.